pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 keywords = ["automatic", "packaging", "publish", "release", "version"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.6"
```

Comment:
It would probably better to leave the [Caret requirement](https://python-poetry.org/docs/dependency-specification/#version-constraints) syntax that was there previously, which allows for Python >= 3.6 but < 4.0. So I would recommend:

    python = "^3.6"

Reply:
Oh, very interesting! I never come across that poetry's docs!